04-onion-routing.md

```diff
@@ -201,6 +201,9 @@ This is formatted according to the Type-Length-Value format defined in [BOLT #1]
     1. type: 16 (`payment_metadata`)
     2. data:
         * [`...*byte`:`payment_metadata`]
+    1. type: 18 (`resolution_format`)
```

Comment:
Intentionally did not use the word failure here, because we may use the same hmac chain format for successes eventually.

Reply:
Reverted back to `fat_error` as the name, no future proofing.